### ISSUE TITLE
Using pretrained EfficientNet can't achieve the accuracy of original paper?

### ISSUE THREAD

**Issue by liudddddz** [rwightman/gen-efficientnet-pytorch#42]
Thanks for your sharing of so many pretrained models.

I am using your EfficientNet-b0 to train on CIFAR-100 using pytorch.My best result was only about 60% acc. I have tried so many hyperparameters and tricks,but still fail to work well.So I have to ask for your kind help. 

Here is my settings:
I used torch.hub to load your pretrained model,and changed out features of last classifier layer to 100.My initial lr was 0.001 used SGD to optimize and reducelronplateau lr scheduler.

It would be great to receive your reply.Have a good day!

**Comment by rwightman** [comment]
As is, these models are not suitable for CIFAR, they are intended for ImageNet sized images scaled from 224x224 and up. You'll have to rescale the CIFAR images to at least 224x224 (which seems kind of pointless). Or you'll need to modify the networks to have a lower output stride... 8 is probably good, using dilations or elemintating some of the blocks.

**Comment by liudddddz** [comment]
Thank you for your kind suggestions.I will have a try
